boards/arm/nucleo_h563zi/nucleo_h563zi-common.dtsi

```diff
@@ -82,7 +82,7 @@
 	clocks = <&pll>;
 	clock-frequency = <DT_FREQ_M(240)>;
 	ahb-prescaler = <1>;
-	apb1-prescaler = <1>;
+	apb1-prescaler = <2>;
```

Comment:
I cannot find any description about this in the reference manual or errata.
Where does this come from (and can it be part of the commit message)?

Reply:
You're right, this is not fully correct, the actual issue is that APB bus is running too fast to accomodate an I2C timings that allows to achieve and I2C clock-frequency as defined by default: `clock-frequency = <I2C_BITRATE_STANDARD>;`

Reply:
Makes sense. It would be nice if you can add it to the commit message :)